solutions/banking/version.tf

```diff
@@ -20,6 +16,10 @@ terraform {
       source  = "Mastercard/restapi"
       version = ">= 1.19.1"
     }
+    shell = {
```

Comment:
we usually rather not depend on 3rd party providers from a security perspective. Is there a fundamental issue with null_resource data + resource that is requiring this provider?

Reply:
Yes - the lack of a GET API for a specific assistant is hindering my efforts to store the current state of the assistant. Null_resource also seems to always want to trigger on apply, so unless I put a lot of work into the script to ensure duplicates aren't created, it won't work - and I would prefer not to rely on the script itself, rather use terraform state. This way, I can also add the assistant integration in the same step rather than relying on an additional data source.